xonsh/ptk_shell/shell.py

```diff
@@ -154,6 +198,7 @@ def __init__(self, **kwargs):
         self.history = ThreadedHistory(PromptToolkitHistory())
 
         ptk_args = {"history": self.history}
+        disable_unwanted_clipboard_actions()
         if HAVE_SYS_CLIPBOARD:
```

Comment:
can we add a configuration option for this? some may want this (people from emacs) :)

Reply:
Good idea - done via `XONSH_COPY_ON_DELETE`